src/benchmark/file_io_write_micro_benchmark.cpp

```diff
@@ -7,55 +7,175 @@
 
 namespace hyrise {
 
-BENCHMARK_DEFINE_F(FileIOWriteMicroBenchmarkFixture, WRITE_NON_ATOMIC)(benchmark::State& state) {
-  auto fd = int32_t{};
-  if ((fd = open(filename, O_WRONLY)) < 0) {
+void write_data_using_write(const size_t from, const size_t to, int32_t fd, uint32_t* data_to_write_start) {
```

Comment:
we need to rename this function to fit the benchmark (to explain, what is write)

Reply:
ok, seeing that below is pwrite, this might be obsolete